<html><head>
<title>SMART: 14_3_3 domain annotation</title>
</head>
<body>
<div>
<div>
<form><input name="keywords" type="text" value="keywords..."/><input type="submit" value="Search SMART"/></form>
</div>
<div>
<img alt="l2"/>
</div>
<div>
<a>SMART MODE:</a>
<p>
<span>NORMAL</span> <a>GENOMIC</a>
</p>
</div>
<a>
<img alt="SMART"/> </a>
<p>
        Letunic et al. (2017) <em>Nucleic Acids Res</em> <a>doi: 10.1093/nar/gkx922</a><em>Nucleic Acids Res</em> <a>doi: 10.1093/nar/gkaa937</a>
</p>
<ul>
<li><a>HOME</a></li>
<li><a>SETUP</a></li>
<li><a>FAQ</a></li>
<li><a>ABOUT</a></li>
<li><a>GLOSSARY</a></li>
<li><a>WHAT'S NEW</a></li>
<li><a>FEEDBACK</a></li>
</ul>
</div>
<div>
<p>There are <a>6285</a> 14_3_3 domains in 6269 proteins in <a>SMART's nrdb</a> database.</p>
<p>Click on the following links for more information.</p>
<a name="annoTable"></a>
<ul>
<li><span>Evolution</span> (species in which this domain is found)</li>
<li>
<div><div><h1>Taxonomic distribution of proteins containing 14_3_3 domain.</h1><p>This tree includes only several representative species. The complete taxonomic breakdown of all proteins with 14_3_3 domain is also <a>avaliable</a>.</p><p><input type="button" value="Expand all nodes"/> <span role="status"></span><input type="text" value="search for a taxonomic node..."/></p></div><div><ul><li><span><span><b>Proteins</b></span><span>%</span><span><b>Domains</b></span><span>%</span></span></li><li><span><a>Bacteria</a><span><a><b>16</b></a></span><span>0.26</span><span><b>16</b></span><span>0.25</span></span><ul><li><span><a>undefined kingdom</a><span><a><b>16</b></a></span><span>0.26</span><span><b>16</b></span><span>0.25</span></span></li></ul></li><li><span><a>Eukaryota</a><span><a><b>6252</b></a></span><span>99.73</span><span><b>6268</b></span><span>99.47</span></span><ul><li><span><a>Fungi</a><span><a><b>1118</b></a></span><span>17.83</span><span><b>1119</b></span><span>17.79</span></span></li><li><span><a>Metazoa</a><span><a><b>2856</b></a></span><span>45.56</span><span><b>2862</b></span><span>45.44</span></span></li><li><span><a>Viridiplantae</a><span><a><b>1766</b></a></span><span>28.17</span><span><b>1766</b></span><span>28.10</span></span></li><li><span><a>undefined kingdom</a><span><a><b>512</b></a></span><span>8.17</span><span><b>521</b></span><span>8.15</span></span></li></ul></li><li><span><a>undefined superkingdom</a><span><a><b>1</b></a></span><span>0.02</span><span><b>1</b></span><span>0.02</span></span><ul><li><span><a>undefined kingdom</a><span><a><b>1</b></a></span><span>0.02</span><span><b>1</b></span><span>0.02</span></span></li></ul></li></ul></div><p>Click on the protein counts, or double click on taxonomic names to display all proteins containing 14_3_3 domain in the selected taxonomic class.</p></div>
</li>
<li><span>Cellular role</span> (predicted cellular role)</li>
<li>
<p><b>Binding / catalysis:</b> Phosphoserine-binding; homodimer formation    
    
    </p></li>
<li><span>Literature</span> (relevant references for this domain)</li><li>
<p>Primary literature is listed below; Automatically-derived, <a>secondary literature</a> is also avaliable.</p>
<div>
<ul>
<li><a>Tsukita S, Yonemura S</a></li>
<li><strong>Cortical actin organization: lessons from ERM (ezrin/radixin/moesin) proteins.</strong></li>
<li>J Biol Chem. 1999; <b>274</b>: 34507-10</li>
</ul>
<ul>
<li><a>Yin HL, Stull JT</a></li>
<li><strong>Proteins that regulate dynamic actin remodeling in response to membrane signaling minireview series.</strong></li>
<li>J Biol Chem. 1999; <b>274</b>: 32529-30</li>
</ul>
<ul>
<li><a>Andrews RK, Harris SJ, McNally T, Berndt MC</a></li>
<li><strong>Binding of purified 14-3-3 zeta signaling protein to discrete amino acid sequences within the cytoplasmic domain of the platelet membrane glycoprotein Ib-IX-V complex.</strong></li>
<li>Biochemistry. 1998; <b>37</b>: 638-47</li>
<li>Display abstract</li>
<li><p>The glycoprotein (GP) Ib-IX-V complex constitutively expressed on the platelet plasma membrane mediates initial adhesion of circulating platelets to vessel wall matrix at high shear, and shear-induced platelet aggregation. In both cases, this involves binding of GP Ib-IX-V to the adhesive glycoprotein, von Willebrand Factor (vWF). vWF binding to GP Ib-IX-V rapidly induces platelet activation, leading to cytoskeletal rearrangement, shape change, and secretion that enables alphaIIbbeta3 integrin (GP IIb-IIIa)-dependent platelet aggregation. All these events are critical in (patho)physiological thrombus formation. The recent discovery that the signaling protein, 14-3-3 zeta, copurifies with the GP Ib-IX complex (minus GP V) [Du, X., Harris, S. J., Tetaz, T. J., Ginsberg, M. H., &amp; Berndt, M. C. (1994) J. Biol. Chem. 269, 18287-18290] indicated a potential mechanism for vWF-dependent signaling. The aim of the present study was to identify discrete amino acid sequences that bind 14-3-3 zeta within the cytoplasmic domain of the receptor. As an initial screening assay, overlapping synthetic peptides based on the cytoplasmic domains of GP Ibalpha (100 residues), GP Ibbeta (34 residues), GP IX (5 residues), and GP V (16 residues) were immobilized and assessed for the ability to bind purified 14-3-3 zeta. The C-terminal sequence GHSL of GP Ibalpha was identified as one 14-3-3 zeta interactive sequence, consistent with previous results [Du, X., Fox, J. E., &amp; Pei, S. (1996) J. Biol. Chem. 271, 7362-7367]. Binding of 125I-labeled 14-3-3 zeta to GHSL-containing peptides was inhibitable by unlabeled 14-3-3 zeta and by anti-14-3-3 zeta IgG. Ala-walking through the GHSL sequence suggested all residues were necessary for optimal binding. In addition, 14-3-3 zeta bound with lower affinity to a peptide based on the central region of the GP Ibalpha cytoplasmic domain (Arg-557-Gly-575), whereas peptide sequences within the cytoplasmic domains of GP Ibbeta (Arg-160-Arg-175) and GP V (Lys-529-Gly-544) bound 14-3-3 zeta with comparable affinity to the GHSL-containing peptide. Soluble GHSL-containing peptides, GP Ibbeta- and GP V-based peptides semidissociated 14-3-3 zeta from GP Ib-IX-V or GP Ib-IX in platelet extracts as analyzed by immunoprecipitation, suggesting these sequences, at least partially, mediate the GP Ib-IX-V-14-3-3 zeta interaction in cells. Further, phosphorylation of the GP Ibbeta peptide at a site corresponding to a protein kinase A phosphorylation site (Ser-166) enhanced the affinity of 14-3-3 zeta binding by approximately 8-fold, suggesting phosphorylation as a potential mechanism for regulating 14-3-3 zeta association with the GP Ib-IX-V complex.</p></li>
</ul>
<ul>
<li><a>Fanger GR, Widmann C, Porter AC, Sather S, Johnson GL, Vaillancourt RR</a></li>
<li><strong>14-3-3 proteins interact with specific MEK kinases.</strong></li>
<li>J Biol Chem. 1998; <b>273</b>: 3476-83</li>
<li>Display abstract</li>
<li><p>MEK (mitogen-activated protein kinase/extracellular signal-regulated kinase kinase) kinases (MEKKs) regulate c-Jun N-terminal kinase and extracellular response kinase pathways. The 14-3-3zeta and 14-3-3epsilon isoforms were isolated in a two-hybrid screen for proteins interacting with the N-terminal regulatory domain of MEKK3. 14-3-3 proteins bound both the N-terminal regulatory and C-terminal kinase domains of MEKK3. The binding affinity of 14-3-3 for the MEKK3 N terminus was 90 nM, demonstrating a high affinity interaction. 14-3-3 proteins also interacted with MEKK1 and MEKK2, but not MEKK4. Endogenous 14-3-3 protein and MEKK1 and MEKK2 were similarly distributed in the cell, consistent with their in vitro interactions. MEKK1 and 14-3-3 proteins colocalized using two-color digital confocal immunofluorescence. Binding of 14-3-3 proteins mapped to the N-terminal 393 residues of 196-kDa MEKK1. Unlike MEKK2 and MEKK3, the C-terminal kinase domain of MEKK1 demonstrated little or no ability to interact with 14-3-3 proteins. MEKK1, but not MEKK2, -3 or -4, is a caspase-3 substrate that when cleaved releases the kinase domain from the N-terminal regulatory domain. Functionally, caspase-3 cleavage of MEKK1 releases the kinase domain from the N-terminal 14-3-3-binding region, demonstrating that caspases can selectively alter protein kinase interactions with regulatory proteins. With regard to MEKK1, -2 and -3, 14-3-3 proteins do not appear to directly influence activity, but rather function as "scaffolds" for protein-protein interactions.</p></li>
</ul>
<ul>
<li><a>Kosaki A, Yamada K, Suga J, Otaka A, Kuzuya H</a></li>
<li><strong>14-3-3beta protein associates with insulin receptor substrate 1 and decreases insulin-stimulated phosphatidylinositol 3'-kinase activity in 3T3L1 adipocytes.</strong></li>
<li>J Biol Chem. 1998; <b>273</b>: 940-4</li>
<li>Display abstract</li>
<li><p>The 14-3-3 protein family has been implicated in growth factor signaling. We investigated whether 14-3-3 protein is involved in insulin signaling in 3T3L1 adipocytes. A significant amount of insulin receptor substrate 1 (IRS-1) was immunodetected in the immunoprecipitate with anti-14-3-3beta antibody at the basal condition. 100 nM insulin increased the amount of IRS-1 in the immunoprecipitate 2.5-fold. The effect of insulin was abolished by 100 nM wortmannin. An in vitro binding study revealed that glutathione S-transferase-14-3-3beta fusion protein directly associates with recombinant IRS-1. Pretreatment of recombinant IRS-1 with alkaline phosphatase clearly decreased this association. Because the recombinant IRS-1 was not phosphorylated on its tyrosine residues, the results suggest that serine/threonine phosphorylation of IRS-1 is responsible for the association. When the cells are treated with insulin, phosphatidylinositol 3'-kinase (PI3K) is supposed to complex either 14-3-3beta-IRS-1 or IRS-1. The 14-3-3beta-IRS-1-PI3K and IRS-1-PI3K complexes were separately prepared by a sequential immunoprecipitation, first with anti-14-3-3beta and then with anti-IRS-1 antibodies. The specific activity of the PI3K in the former was approximately half of that in the latter, suggesting that 14-3-3beta protein bound to IRS-1 inhibits insulin-stimulated lipid kinase activity of PI3K in 3T3L1 adipocytes.</p></li>
</ul>
<ul>
<li><a>Shaw RJ, Henry M, Solomon F, Jacks T</a></li>
<li><strong>RhoA-dependent phosphorylation and relocalization of ERM proteins into apical membrane/actin protrusions in fibroblasts.</strong></li>
<li>Mol Biol Cell. 1998; <b>9</b>: 403-19</li>
<li>Display abstract</li>
<li><p>The ERM proteins (ezrin, radixin, and moesin) are a group of band 4. 1-related proteins that are proposed to function as membrane/cytoskeletal linkers. Previous biochemical studies have implicated RhoA in regulating the association of ERM proteins with their membrane targets. However, the specific effect and mechanism of action of this regulation is unclear. We show that lysophosphatidic acid stimulation of serum-starved NIH3T3 cells resulted in relocalization of radixin into apical membrane/actin protrusions, which was blocked by inactivation of Rho by C3 transferase. An activated allele of RhoA, but not Rac or CDC42Hs, was sufficient to induce apical membrane/actin protrusions and localize radixin or moesin into these structures in both Rat1 and NIH3T3 cells. Lysophosphatidic acid treatment led to phosphorylation of radixin preceding its redistribution into apical protrusions. Significantly, cotransfection of RhoAV14 or C3 transferase with radixin and moesin revealed that RhoA activity is necessary and sufficient for their phosphorylation. These findings reveal a novel function of RhoA in reorganizing the apical actin cytoskeleton and suggest that this function may be mediated through phosphorylation of ERM proteins.</p></li>
</ul>
<ul>
<li><a>Xu HM, Gutmann DH</a></li>
<li><strong>Merlin differentially associates with the microtubule and actin cytoskeleton.</strong></li>
<li>J Neurosci Res. 1998; <b>51</b>: 403-15</li>
<li>Display abstract</li>
<li><p>The neurofibromatosis 2 (NF2) suppressor gene encodes a protein termed merlin (or schwannomin) with sequence similarity to a family of proteins that link the actin cytoskeleton to cell surface glycoproteins. Members of this ERM family of proteins include ezrin, radixin, and moesin. These proteins contain a carboxyl (C-) terminus actin binding site. In contrast to the ERM proteins, merlin lacks the conventional C-terminal actin binding site, but still localizes to the ruffling edge of plasma membranes. In this study, we investigate the ability of merlin to interact with actin through a nonconventional actin binding domain. We demonstrate for the first time that merlin can associate with polymerized actin in vitro by virtue of an amino (N-) terminal actin binding domain including residues 178-367. Merlin actin binding is not affected by several naturally-occurring NF2 patient mutations or alternatively spliced isoforms. These results suggest that merlin, like other ERM proteins, can directly interact with the actin cytoskeleton. In addition, merlin associates with polymerized microtubules in vitro using a novel microtubule binding region in the N-terminal region of merlin that is masked in the full-length merlin molecule, such that wild-type functional merlin in the "closed" conformation fails to bind polymerized microtubules. These microtubule association results confirm the notion that merlin exists in "open" and "closed" conformations relevant to its function as a negative growth regulator.</p></li>
</ul>
<ul>
<li><a>Ichimura T <i>et al.</i></a></li>
<li><strong>The 14-3-3 protein binds its target proteins with a common site located towards the C-terminus.</strong></li>
<li>FEBS Lett. 1997; <b>413</b>: 273-6</li>
<li>Display abstract</li>
<li><p>The 14-3-3 protein family binds a variety of proteins in cell-signaling pathways, but the structural elements necessary for the ligand binding are poorly understood. Here we demonstrate that the 'box-1' region, which spans residues 171-213 in the eta-isoform and was previously identified as the binding site of 14-3-3 to the phosphorylated tryptophan hydroxylase, plays a critical role in the interaction with many target proteins. Using a series of truncated 14-3-3 mutants, we show that the mutant 167-213 carrying box-1 binds bacurovirus-expressed Raf-1 and Bcr protein kinases to the similar extent as the full-length 14-3-3 in a phosphorylation-dependent manner, while the mutants lacking this region abolish the binding activity. Furthermore, the box-1 region also appears essential for binding of 14-3-3 to more than 40 phosphoproteins found in the brainstem extract. These results suggest that the box-1 region, consisting of helices 7 and 8 in the tertiary structure, is a common structural element whereby the 14-3-3 protein binds many, if not all, target proteins.</p></li>
</ul>
<ul>
<li><a>Yaffe MB <i>et al.</i></a></li>
<li><strong>The structural basis for 14-3-3:phosphopeptide binding specificity.</strong></li>
<li>Cell. 1997; <b>91</b>: 961-71</li>
<li>Display abstract</li>
<li><p>The 14-3-3 family of proteins mediates signal transduction by binding to phosphoserine-containing proteins. Using phosphoserine-oriented peptide libraries to probe all mammalian and yeast 14-3-3s, we identified two different binding motifs, RSXpSXP and RXY/FXpSXP, present in nearly all known 14-3-3 binding proteins. The crystal structure of 14-3-3zeta complexed with the phosphoserine motif in polyoma middle-T was determined to 2.6 A resolution. The bound peptide is in an extended conformation, with a tight turn created by the pS +2 Pro in a cis conformation. Sites of peptide-protein interaction in the complex rationalize the peptide library results. Finally, we show that the 14-3-3 dimer binds tightly to single molecules containing tandem repeats of phosphoserine motifs, implicating bidentate association as a signaling mechanism with molecules such as Raf, BAD, and Cbl.</p></li>
</ul>
<ul>
<li><a>Jin DY, Lyu MS, Kozak CA, Jeang KT</a></li>
<li><strong>Function of 14-3-3 proteins.</strong></li>
<li>Nature. 1996; <b>382</b>: 308-308</li>
</ul>
<ul>
<li><a>Muslin AJ, Tanner JW, Allen PM, Shaw AS</a></li>
<li><strong>Interaction of 14-3-3 with signaling proteins is mediated by the recognition of phosphoserine.</strong></li>
<li>Cell. 1996; <b>84</b>: 889-97</li>
<li>Display abstract</li>
<li><p>The highly conserved and ubiquitously expressed 14-3-3 family of proteins bind to a variety of proteins involved in signal transduction and cell cycle regulation. The nature and specificity of 14-3-3 binding is, however, not known. Here we show that 14-3-3 is a specific phosphoserine-binding protein. Using a panel of phosphorylated peptides based on Raf-1, we have defined the 14-3-3 binding motif and show that most of the known 14-3-3 binding proteins contain the motif. Peptides containing the motif could disrupt 14-3-3 complexes and inhibit maturation of Xenopus laevis oocytes. These results suggest that the interactions of 14-3-3 with signaling proteins are critical for the activation of signaling proteins. Our findings also suggest novel roles for serine/threonine phosphorylation in the assembly of protein-protein complexes.</p></li>
</ul>
<ul>
<li><a>Nakamura H, Ozawa H</a></li>
<li><strong>Immunolocalization of CD44 and the ERM family in bone cells of mouse tibiae.</strong></li>
<li>J Bone Miner Res. 1996; <b>11</b>: 1715-22</li>
<li>Display abstract</li>
<li><p>We studied the immunohistochemical localization of CD44, hyaluronate receptor, and the ezrin-radixin-moesin (ERM) family, actin binding proteins, in bone cells using confocal laser scanning microscopy and transmission electron microscopy to clarify the mechanism of the organization of their cytoskeletons. In osteoclasts, intense immunoreactivity to CD44 could be detected on their basolateral plasma membranes. There was less reactivity observed in the area of the plasma membrane in direct contact with the bone surface. The immunogold electron-microscopical method revealed that CD44 was mainly localized on the microvilli of the basolateral plasma membrane. The plasma membrane of the clear zone and the ruffled border were not immunolabeled with CD44. As for the ERM family, the basolateral plasma membrane of osteoclasts was stained with antimoesin monoclonal antibody, but not with ezrin or radixin. In osteoblasts attached to the bone surface, immunoreactivity to CD44 was restricted to their cytoplasmic processes. They showed immunoreactivities to radixin and moesin on the cytoplasmic side of their plasma membrane when in contact with each other. However, although osteocytes in the bone matrix demonstrate an intense immunolabeling with CD44 on their plasma membrane, they scarcely show immunoreactivity to the ERM family. These findings suggest that: (1) the CD44-moesin-actin filament system is involved in the organization of cytoskeletons in the basolateral plasma membrane of osteoclasts; and (2) other mechanisms, rather than the CD44 and the ERM family, may be involved in the cells of osteoblast lineage.</p></li>
</ul>
<ul>
<li><a>Wang W, Shakes DC</a></li>
<li><strong>Molecular evolution of the 14-3-3 protein family.</strong></li>
<li>J Mol Evol. 1996; <b>43</b>: 384-98</li>
<li>Display abstract</li>
<li><p>Members of the highly conserved and ubiquitous 14-3-3 protein family modulate a wide variety of cellular processes. To determine the evolutionary relationships among specific 14-3-3 proteins in different plant, animal, and fungal species and to initiate a predictive analysis of isoform-specific differences in light of the latest functional and structural studies of 14-3-3, multiple alignments were constructed from forty-six 14-3-3 sequences retrieved from the GenBank and SwissProt databases and a newly identified second 14-3-3 gene from Caenorhabditis elegans. The alignment revealed five highly conserved sequence blocks. Blocks 2-5 correlate well with the alpha helices 3, 5, 7, and 9 which form the proposed internal binding domain in the three-dimensional structure model of the functioning dimer. Amino acid differences within the functional and structural domains of plant and animal 14-3-3 proteins were identified which may account for functional diversity amongst isoforms. Protein phylogenic trees were constructed using both the maximum parsimony and neighbor joining methods of the PHYLIP(3.5c) package; 14-3-3 proteins from Entamoeba histolytica, an amitochondrial protozoa, were employed as an outgroup in our analysis. Epsilon isoforms from the animal lineage form a distinct grouping in both trees, which suggests an early divergence from the other animal isoforms. Epsilons were found to be more similar to yeast and plant isoforms than other animal isoforms at numerous amino acid positions, and thus epsilon may have retained functional characteristics of the ancestral protein. The known invertebrate proteins group with the nonepsilon mammalian isoforms. Most of the current 14-3-3 isoform diversity probably arose through independent duplication events after the divergence of the major eukaryotic kingdoms. Divergence of the seven mammalian isoforms beta, zeta, gamma, eta, epsilon, tau, and sigma (stratifin/HME1) occurred before the divergence of mammalian and perhaps before the divergence of vertebrate species. A possible ancestral 14-3-3 sequence is proposed.</p></li>
</ul>
<ul>
<li><a>Zigmond SH</a></li>
<li><strong>Signal transduction and actin filament organization.</strong></li>
<li>Curr Opin Cell Biol. 1996; <b>8</b>: 66-73</li>
<li>Display abstract</li>
<li><p>Small GTP-binding proteins of the Rho family appear to integrate extracellular signals from diverse receptor types and initiate rearrangements of F-actin. Active members of the Rho family, Rho and Rac, are now joined by Cdc42 which induces filopodia. Downstream of the Rho family proteins, actin polymerization may be induced by an increase in the availability of actin filament barbed ends. Actin organization may be affected by exposure of actin-binding sites on proteins such as vinculin and ezrin.</p></li>
</ul>
<ul>
<li><a>Cantiello HF</a></li>
<li><strong>Role of the actin cytoskeleton on epithelial Na+ channel regulation.</strong></li>
<li>Kidney Int. 1995; <b>48</b>: 970-84</li>
<li>Display abstract</li>
<li><p>The regulatory role of actin filament organization on epithelial Na+ channel activity is reviewed in this report. The actin cytoskeleton, consisting of actin filaments and associated actin-binding proteins, is essential to various cellular events including the maintenance of cell shape, the onset of cell motility, and the distribution and stability of integral membrane proteins. Functional interactions between the actin cytoskeleton and specific membrane transport proteins are, however, not as well understood. Recent studies from our laboratory have determined that dynamic changes in the actin cytoskeletal organization may represent a novel signaling mechanism in the regulation of ion transport in epithelia. This report summarizes work conducted in our laboratory leading to an understanding of the molecular steps associated with the regulatory role of the actin-based cytoskeleton on epithelial Na+ channel function. The basis of this interaction lies on the regulation by actin-binding proteins and adjacent structures, of actin filament organization which in turn, modulates ion channel activity. The scope of this interaction may extend to such relevant cellular events as the vasopressin response in the kidney.</p></li>
</ul>
<ul>
<li><a>Friederich E, Gouin E, Hellio R, Kocks C, Cossart P, Louvard D</a></li>
<li><strong>Targeting of Listeria monocytogenes ActA protein to the plasma membrane as a tool to dissect both actin-based cell morphogenesis and ActA function.</strong></li>
<li>EMBO J. 1995; <b>14</b>: 2731-44</li>
<li>Display abstract</li>
<li><p>Actin assembly on the surface of Listeria monocytogenes in the cytoplasm of infected cells provides a model to study actin-based motility and changes in cell shape. We have shown previously that the ActA protein, exposed on the bacterial surface, is required for polarized nucleation of actin filaments. To investigate whether plasma membrane-associated ActA can control the organization of microfilaments and cell shape, variants of ActA, in which the bacterial membrane signal had been replaced by a plasma membrane anchor sequence, were produced in mammalian cells. While both cytoplasmic and membrane-bound forms of ActA increased the F-actin content, only membrane-associated ActA caused the formation of plasma membrane extensions. This finding suggests that ActA acts as an actin filament nucleator and shows that permanent association with the inner face of the plasma membrane is required for changes in cell shape. Based on the observation that the amino-terminal segment of ActA and the remaining portion which includes the proline-rich repeats cause distinct phenotypic modifications in transfected cells, we propose a model in which two functional domains of ActA cooperate in the nucleation and dynamic turnover of actin filaments. The present approach is a new model system to dissect the mechanism of action of ActA and to further investigate interactions of the plasma membrane and the actin cytoskeleton during dynamic changes of cell shape.</p></li>
</ul>
<ul>
<li><a>Liu D, Bienkowska J, Petosa C, Collier RJ, Fu H, Liddington R</a></li>
<li><strong>Crystal structure of the zeta isoform of the 14-3-3 protein.</strong></li>
<li>Nature. 1995; <b>376</b>: 191-4</li>
<li>Display abstract</li>
<li><p>The 14-3-3 family of proteins have recently been identified as regulatory elements in intracellular signalling pathways: 14-3-3 proteins bind to oncogene and proto-oncogene products, including c-Raf-1 (refs 2-5), c-Bcr (ref. 6) and polyomavirus middle-T antigen; overexpression of 14-3-3 activates Raf kinase in yeast and induces meiotic maturation in Xenopus oocytes. Here we report the crystal structure of the major isoform of mammalian 14-3-3 proteins at 2.9 A resolution. Each subunit of the dimeric protein consists of a bundle of nine antiparallel helices that form a palisade around an amphipathic groove. The groove is large enough to accommodate a tenth helix, and we propose that binding to an amphipathic helix represents a general mechanism for the interaction of 14-3-3 with diverse cellular proteins. The residues in the dimer interface and the putative ligand-binding surface are invariant among vertebrates, yeast and plants, suggesting a conservation of structure and function throughout the 14-3-3 family.</p></li>
</ul>
<ul>
<li><a>Xiao B <i>et al.</i></a></li>
<li><strong>Structure of a 14-3-3 protein and implications for coordination of multiple signalling pathways.</strong></li>
<li>Nature. 1995; <b>376</b>: 188-91</li>
<li>Display abstract</li>
<li><p>A broad range of organisms and tissues contain 14-3-3 proteins, which have been associated with many diverse functions including critical roles in signal transduction pathways, exocytosis and cell cycle regulation. We report here the crystal structure of the human T-cell 14-3-3 isoform (tau) dimer at 2.6 A resolution. Each monomer (Mr 28K) is composed of an unusual arrangement of nine antiparallel alpha-helices organized as two structural domains. The dimer creates a large, negatively charged channel approximately 35 A broad, 35 A wide and 20 A deep. Overall, invariant residues line the interior of this channel whereas the more variable residues are distributed on the outer surface. At the base of this channel is a 16-residue segment of 14-3-3 which has been implicated in the binding of 14-3-3 to protein kinase C.</p></li>
</ul>
<ul>
<li><a>Hitt AL, Luna EJ</a></li>
<li><strong>Membrane interactions with the actin cytoskeleton.</strong></li>
<li>Curr Opin Cell Biol. 1994; <b>6</b>: 120-30</li>
<li>Display abstract</li>
<li><p>Recent advances have been made in our understanding of the direct binding of actin to integral membrane proteins. New information has been obtained about indirect actin-membrane associations through spectrin superfamily members and through proteins at the cytoplasmic surfaces of focal contacts and adherens junctions.</p></li>
</ul>
<ul>
<li><a>Goodlad GA, Clark CM</a></li>
<li><strong>Actin and actin-binding proteins in plasma membranes derived from Walker 256 ascites or solid tumour cells.</strong></li>
<li>Biochim Biophys Acta. 1993; <b>1145</b>: 177-9</li>
<li>Display abstract</li>
<li><p>Plasma membranes from Walker 256 carcinoma cells grown ascitically or as a solid tumour were examined with respect to actin content, [3H]cytochalasin B-binding and the binding of 125I-labelled G-actin to membrane proteins separated by SDS-PAGE. Differences were observed both in cytochalasin B-binding to membrane actin and affinity of 125I-labelled G-actin for specific membrane proteins.</p></li>
</ul>
<ul>
<li><a>Furuhashi K, Inagaki M, Hatano S</a></li>
<li><strong>[Functional regulation of cytoskeleton by phosphorylation: actin phosphorylation]</strong></li>
<li>Seikagaku. 1992; <b>64</b>: 255-9</li>
</ul>
<ul>
<li><a>Nairn AC, Aderem A</a></li>
<li><strong>Calmodulin and protein kinase C cross-talk: the MARCKS protein is an actin filament and plasma membrane cross-linking protein regulated by protein kinase C phosphorylation and by calmodulin.</strong></li>
<li>Ciba Found Symp. 1992; <b>164</b>: 145-54</li>
<li>Display abstract</li>
<li><p>The myristoylated, alanine-rich C kinase (PKC) substrate (MARCKS) is a major, specific substrate of PKC that is phosphorylated during macrophage and neutrophil activation, growth factor-dependent mitogenesis and neurosecretion. MARCKS is also a calmodulin-binding protein and binding of calmodulin inhibits phosphorylation of the protein by PKC. Several recent observations from our laboratories suggest a role for MARCKS in cellular morphology and motility. First, in macrophages MARCKS is located at points of cellular adherence where actin filaments insert at the plasma membrane and is released to the cytoplasm upon activation of PKC. Second, during neutrophil chemotaxis MARCKS undergoes a cycle of release from, and reassociation with, the plasma membrane. Third, in vitro, MARCKS is an F-actin cross-linking protein whose activity is inhibited by PKC-mediated phosphorylation and by binding to calmodulin. MARCKS therefore appears to be a regulated cross-bridge between actin and the plasma membrane. Regulation of the plasma membrane-binding and actin-binding properties of MARCKS represents a convergence of the PKC and calmodulin signal transduction pathways in the control of actin cytoskeleton-plasma membrane interactions.</p></li>
</ul>
<ul>
<li><a>Hartwig JH, Kwiatkowski DJ</a></li>
<li><strong>Actin-binding proteins.</strong></li>
<li>Curr Opin Cell Biol. 1991; <b>3</b>: 87-97</li>
<li>Display abstract</li>
<li><p>Much new information on the sequence, structure, and function of filament crosslinking, capping, and severing proteins is now known. Other significant findings include identification of a new abundant monomer-sequestering protein in platelets, and evidence that many actin-binding proteins interact with phosphoinositides and that this interaction may have metabolic consequences.</p></li>
</ul>
<ul>
<li><a>Pershina VP, Pinaev GP, Goncharova EI</a></li>
<li><strong>[Actin interaction with the plasma membrane fraction of the cells from a suspension subline of murine L fibroblasts]</strong></li>
<li>Tsitologiia. 1990; <b>32</b>: 1198-204</li>
<li>Display abstract</li>
<li><p>Effect of plasma membranes of murine fibroblasts cultivated in suspension on actin polymerization was studied. Using low shear viscometry of actin-membrane mixtures together with the number of extractions of membranes with actin depolymerizing buffers it was found that at least two polypeptides 220 and 94 kDa may be involved into the actin filaments-plasma membrane interaction.</p></li>
</ul>
<ul>
<li><a>Shariff A, Luna EJ</a></li>
<li><strong>Dictyostelium discoideum plasma membranes contain an actin-nucleating activity that requires ponticulin, an integral membrane glycoprotein.</strong></li>
<li>J Cell Biol. 1990; <b>110</b>: 681-92</li>
<li>Display abstract</li>
<li><p>In previous equilibrium binding studies, Dictyostelium discoideum plasma membranes have been shown to bind actin and to recruit actin into filaments at the membrane surface. However, little is known about the kinetic pathway(s) through which actin assembles at these, or other, membranes. We have used actin fluorescently labeled with N-(1-pyrenyl)iodoacetamide to examine the kinetics of actin assembly in the presence of D. discoideum plasma membranes. We find that these membranes increase the rate of actin polymerization. The rate of membrane-mediated actin polymerization is linearly dependent on membrane protein concentrations up to 20 micrograms/ml. Nucleation (the association of activated actin monomers into oligomers) appears to be the primary step of polymerization that is accelerated. A sole effect on the initial salt-induced actin conformational change (activation) is ruled out because membranes accelerate the polymerization of pre-activated actin as well as actin activated in the presence of membranes. Elongation of preexisting filaments also is not the major step of polymerization facilitated by membranes since membranes stripped of all peripheral components, including actin, increase the rate of actin assembly to about the same extent as do membranes containing small amounts of endogenous actin. Acceleration of the nucleation step by membranes also is supported by an analysis of the dependence of polymerization lag time on actin concentration. The barbed ends of membrane-induced actin nuclei are not obstructed by the membranes because the barbed end blocking agent, cytochalasin D, reduces the rate of membrane-mediated actin nucleation. Similarly, the pointed ends of the nuclei are not blocked by membranes since the depolymerization rate of gelsolin-capped actin is unchanged in the presence of membranes. These results are consistent with previous observations of lateral interactions between membranes and actin filaments. These results also are consistent with two predictions from a model based on equilibrium binding studies; i.e., that plasma membranes should nucleate actin assembly and that membrane-bound actin nuclei should have both ends free (Schwartz, M. A., and E. J. Luna. 1988. J. Cell Biol. 107:201-209). Integral membrane proteins mediate the actin nucleation activity because activity is eliminated by heat denaturation, treatment with reducing agents, or proteolysis of membranes. Activity also is abolished by solubilization with octylglucoside but is reconstituted upon removal or dilution of the detergent. Ponticulin, the major actin-binding protein in plasma membranes, appears to be necessary for nucleation activity since activity is not reconstituted from detergent extracts depleted of ponticulin.</p></li>
</ul>
<ul>
<li><a>Keresztes M</a></li>
<li><strong>Three actin-binding proteins are developmentally regulated in rat liver plasma membranes.</strong></li>
<li>FEBS Lett. 1989; <b>249</b>: 51-5</li>
<li>Display abstract</li>
<li><p>Actin-binding membrane proteins (linking microfilaments to the cell membrane) are involved in cytoskeleton-membrane interactions which are supposed to undergo profound changes during cell proliferation and development. In this study 8 polypeptides were shown to bind F-actin directly in the liver cell membranes of mature rats. From these, the abundance of three polypeptides, of 130, 50 and 36 kDa, was observed to increase considerably during postnatal development, which indicates a developmental change in the cytoskeleton-membrane interactions.</p></li>
</ul>
<ul>
<li><a>Miller KG, Alberts BM</a></li>
<li><strong>F-actin affinity chromatography: technique for isolating previously unidentified actin-binding proteins.</strong></li>
<li>Proc Natl Acad Sci U S A. 1989; <b>86</b>: 4808-12</li>
<li>Display abstract</li>
<li><p>We have developed stable and easy to use filamentous actin (F-actin) affinity-chromatography columns that selectively purify proteins that bind to actin filaments from cell extracts. Most traditional assays for actin-associated proteins screen for their effects on actin polymerization or actin filament crosslinking. Because our technique requires only actin-filament binding, it can identify additional types of proteins involved in the function of the actin cytoskeleton. By chromatographing extracts of several types of cells on these columns, we show that known actin-binding proteins are selectively retained as a subset of a larger group of actin-binding proteins that have not been identified previously.</p></li>
</ul>
<ul>
<li><a>St-Onge D, Gicquaud C</a></li>
<li><strong>Evidence of direct interaction between actin and membrane lipids.</strong></li>
<li>Biochem Cell Biol. 1989; <b>67</b>: 297-300</li>
<li>Display abstract</li>
<li><p>Actin is a protein component of the cystoskeleton and is involved in cell motility. It is believed generally that actin filaments are attached to the cell membrane through an interaction with membranous actin-binding proteins. By using an in vitro system composed of liposomes and actin, we have shown that actin may also interact directly with the phospholipids of the membrane. Actin deposited at the surface of the liposome is organized in two regular patterns: a paracrystalline sheet of parallel filaments in register, or a netlike organization. These interactions of actin with membrane lipids occur only in the presence of millimolar concentrations of Mg2+. These results suggest that the interaction of the cytoskeleton with the membrane involves, at least in part, a direct association of actin with phospholipids.</p></li>
</ul>
<ul>
<li><a>Tranter MP, Sugrue SP, Schwartz MA</a></li>
<li><strong>Evidence for a direct, nucleotide-sensitive interaction between actin and liver cell membranes.</strong></li>
<li>J Cell Biol. 1989; <b>109</b>: 2833-40</li>
<li>Display abstract</li>
<li><p>We have investigated the association of actin with membranes isolated from rat liver. A plasma membrane-enriched fraction prepared by homogenization in a low salt/CaCl2 buffer was found to contain a substantial amount of residual actin which could be removed by treatment with 1 M Na2CO3/NaHCO3, pH 10.5. Using a sedimentation binding assay that uses gelsolin to shorten actin filaments and render membrane binding saturable (Schwartz, M. A., and E. J. Luna. 1986. J. Cell Biol. 102:2067-2075), we found that membranes stripped of endogenous actin bound 125I-actin in a specific and saturable manner. Scatchard plots of binding data were linear, indicating a single class of binding sites with a Kd of 1.6 microns; 66 micrograms actin bound/mg membrane protein at saturation. Binding of actin to liver cell membranes was negligible with unstripped membranes, was competed by excess unlabeled actin, and was greatly reduced by preheating or proteolytic digestion of the membranes. Kinetic measurements showed that binding had an initial lag phase and was strongly temperature dependent. The binding of actin to liver cell membranes was also found to be competitively inhibited by ATP and other nucleotides, including the nonhydrolyzable analogue AMP-PNP. We conclude that we have reconstituted an interaction between actin and integral membrane proteins from the rat liver. This interaction exhibits a number of distinctive features which have not been observed in other actin-membrane systems.</p></li>
</ul>
<ul>
<li><a>Pershina VP, Pinaev GP, Semenova EG</a></li>
<li><strong>[Actin interaction with the plasma membrane fraction of cells of murine fibroblast L sublines differing by growth type in culture]</strong></li>
<li>Tsitologiia. 1988; <b>30</b>: 582-8</li>
<li>Display abstract</li>
<li><p>The association between murine fibroblast L plasma membranes and actin was studied by means of low-shear viscometry of membrane-actin mixtures. Membrane fractions of 3 genetically related sublines of L cells were used differing in cytoskeleton structural organization. The suspension cell subline LS membranes showed actin gelatin activity. On the contrary, the membranes of monolayer cell sublines L-929 and LSM were seen to bind actin and to cause decrease in viscosity. Possible mechanisms of such interactions are discussed.</p></li>
</ul>
<ul>
<li><a>Brown SS, Petzold AS</a></li>
<li><strong>Using antibodies against Dictyostelium membranes to identify an actin-binding membrane protein.</strong></li>
<li>J Cell Biol. 1987; <b>104</b>: 513-8</li>
<li>Display abstract</li>
<li><p>Polyclonal antibodies made against Dictyostelium discoideum membranes were used to block the interaction of those membranes with actin. As expected, actin interacted mostly with the internal surface of the membrane, demonstrated by the fact that whole cells could only absorb out a minor fraction of the blocking antibody. The antibody was used to show that the membrane component(s) which interacted with actin were probably integral; they could be extracted with detergent but not with solutions designed to extract peripheral membrane proteins. To identify the responsible protein(s), Western transfers of membranes were cut into fractions which were tested for their ability to absorb out the blocking activity of the antibody. We observed a single peak at a molecular weight of approximately 20,000, and thus conclude that a 20,000-mol-wt protein is a major integral membrane actin-binding protein in Dictyostelium. This approach to the identification of proteins involved in actin-membrane interaction has allowed us to make the first identification of an actin-binding membrane protein which is based on its activity in native membranes.</p></li>
</ul>
<ul>
<li><a>Barmann M, Wadsack J, Frimmer M</a></li>
<li><strong>A 50 kDa, actin-binding protein in plasma membranes of rat hepatocytes and of rat liver tumors.</strong></li>
<li>Biochim Biophys Acta. 1986; <b>859</b>: 110-6</li>
<li>Display abstract</li>
<li><p>Plasma membranes from normal rat livers and rat liver tumors were compared by SDS-gel electrophoresis, and analyzed for actin-binding proteins by an 125I-labelled actin gel-overlay assay and by actin-affinity blotting. After treatment of rats with alpha-hexachlorocyclohexane and after induction of liver tumors by combined treatment with N-nitrosomorpholine and phenobarbital, liver plasma membranes prepared from these animals were found to be highly enriched in an actin-binding, 50 kDa polypeptide. This polypeptide seemed to be an integral protein of the plasma membrane as judged by Triton X-114-phase separation. Microsomes did not contain an actin-binding polypeptide in the 50 kDa region. Therefore, the 50 kDa protein is a candidate for interaction of actin with the liver cell plasma membrane. A possible relationship of this protein with the multi-specific, cholate transporting system of the rat liver plasma membrane is discussed.</p></li>
</ul>
<ul>
<li><a>Mooseker MS, Coleman TR, Conzelman KA</a></li>
<li><strong>Calcium and the regulation of cytoskeletal assembly, structure and contractility.</strong></li>
<li>Ciba Found Symp. 1986; <b>122</b>: 232-49</li>
<li>Display abstract</li>
<li><p>Calcium plays a central role in the regulation of cytoskeletal assembly, structure and contractility. In the case of actin there are a number of functional classes of actin-binding proteins which confer on a given actin filament its specific function in the cell. Among these various classes of actin-binding proteins are a subset of proteins whose activity is either regulated directly or indirectly (for example, through calmodulin) by Ca2+. This includes the regulation of actin-myosin interaction, actin assembly, actin filament interaction and the formation of supramolecular cytoskeletal networks, and the interaction of actin with membranes. Examples of these various modes of Ca2+-dependent regulation of cytoskeletal structure and contractility are discussed.</p></li>
</ul>
<ul>
<li><a>Schwartz MA, Luna EJ</a></li>
<li><strong>Binding and assembly of actin filaments by plasma membranes from Dictyostelium discoideum.</strong></li>
<li>J Cell Biol. 1986; <b>102</b>: 2067-75</li>
<li>Display abstract</li>
<li><p>The binding of native, 125I-Bolton-Hunter-labeled actin to purified Dictyostelium discoideum plasma membranes was measured using a sedimentation assay. Binding was saturable only in the presence of the actin capping protein, gelsolin. In the presence of gelsolin, the amount of actin bound at saturation to three different membrane preparations was 80, 120, and 200 micrograms/mg of membrane protein. The respective concentrations of actin at half-saturation were 8, 12, and 18 micrograms/ml. The binding curves were sigmoidal, indicating positive cooperativity at low actin concentrations. This cooperativity appeared to be due to actin-actin associations during polymerization, since phalloidin converted the curve to a hyperbolic shape. In kinetic experiments, actin added as monomers bound to membranes at a rate of 0.6 microgram ml-1 min-1, while pre-polymerized actin bound at a rate of 3.0 micrograms ml-1 min-1. Even in the absence of phalloidin, actin bound to membranes at concentrations well below the normal critical concentration. This membrane-bound actin stained with rhodamine-phalloidin and was cross-linked by m-maleimidobenzoyl succinimide ester, a bifunctional cross-linker, into multimers with the same pattern observed for cross-linked F-actin. We conclude that D. discoideum plasma membranes bind actin specifically and saturably and that these membranes organize actin into filaments below the normal critical concentration for polymerization. This interaction probably occurs between multiple binding sites on the membrane and the side of the actin filament, and may be related to the clustering of membrane proteins.</p></li>
</ul>
<ul>
<li><a>Yamamoto K</a></li>
<li><strong>The binding of skeletal muscle C-protein to regulated actin.</strong></li>
<li>FEBS Lett. 1986; <b>208</b>: 123-7</li>
<li>Display abstract</li>
<li><p>The binding of C-protein, a component of thick filament of myofibrils, to regulated actin filaments in the presence or absence of Ca2+ was studied. The amount of C-protein bound to regulated actin filaments in the presence of Ca2+ was higher than those in the absence of Ca2+. The addition of C-protein to regulated actin caused an increase in turbidity, especially in the presence of Ca2+, and this was found to result from side-by-side association of actin filaments into bundles. In the absence of Ca2+, no actin filament bundles were formed.</p></li>
</ul>
<ul>
<li><a>Brown SS</a></li>
<li><strong>The linkage of actin to non-erythroid membranes.</strong></li>
<li>Bioessays. 1985; <b>3</b>: 65-7</li>
</ul>
<ul>
<li><a>Fox JE</a></li>
<li><strong>Identification of actin-binding protein as the protein linking the membrane skeleton to glycoproteins on platelet plasma membranes.</strong></li>
<li>J Biol Chem. 1985; <b>260</b>: 11970-7</li>
<li>Display abstract</li>
<li><p>Platelets have previously been shown to contain a membrane skeleton that is composed of actin filaments, actin-binding protein, and three membrane glycoproteins (GP), GP Ib, GP Ia, and a minor glycoprotein of Mr = 250,000. The present study was designed to determine how the membrane glycoproteins were linked to actin filaments. Unstimulated platelets were lysed with Triton X-100, and the membrane skeleton was isolated on sucrose density gradients or by high-speed centrifugation. The association of the membrane glycoproteins with the actin filaments was disrupted when actin-binding protein was hydrolyzed by activity of the Ca2+-dependent protease, which was active in platelet lysates upon addition of Ca2+ in the absence of leupeptin. Similarly, activation of the Ca2+-dependent protease in intact platelets by the addition of a platelet agonist also caused the membrane glycoproteins to dissociate from the membrane skeleton. Affinity-purified actin-binding protein antibodies immunoprecipitated the membrane glycoproteins from platelet lysates in which actin filaments had been removed by DNase I-induced depolymerization and high-speed centrifugation. These results demonstrate that actin-binding protein links actin filaments of the platelet membrane skeleton to three plasma membrane glycoproteins and that filaments are released from their attachment site when actin-binding protein is hydrolyzed by the Ca2+-dependent protease within intact platelets during platelet activation.</p></li>
</ul>
<ul>
<li><a>Sutoh K</a></li>
<li><strong>[Interactions between actin and actin-binding proteins]</strong></li>
<li>Tanpakushitsu Kakusan Koso. 1985; <b>30</b>: 1290-300</li>
</ul>
<ul>
<li><a>Goodloe-Holland CM, Luna EJ</a></li>
<li><strong>A membrane cytoskeleton from Dictyostelium discoideum. III. Plasma membrane fragments bind predominantly to the sides of actin filaments.</strong></li>
<li>J Cell Biol. 1984; <b>99</b>: 71-8</li>
<li>Display abstract</li>
<li><p>The binding between sonicated Dictyostelium discoideum plasma membrane fragments and F-actin on Sephacryl S-1000 beads was found to be competitively inhibited by myosin subfragment-1. This inhibition is MgATP-sensitive, exhibits a Ki of approximately 5 X 10(-8) M, and is reciprocal, since membranes inhibit the binding of 125I-heavy meromyosin to F-actin on beads. These experiments demonstrate that membrane binding and S-1 binding to F-actin on beads are mutually exclusive and, therefore, that the membrane fragments bind predominantly to the sides, rather than to the ends, of the actin filaments. This conclusion is supported by electron micrographs that show many lateral associations between membrane fragments and bead-associated actin filaments. Such lateral associations could play an important role in the organization and lateral movement of membrane proteins by the cytomusculature.</p></li>
</ul>
<ul>
<li><a>Therien HM, Gruda J, Carrier F</a></li>
<li><strong>Interaction of filamentous actin with isolated liver plasma membranes.</strong></li>
<li>Eur J Cell Biol. 1984; <b>35</b>: 112-21</li>
<li>Display abstract</li>
<li><p>Actin-membrane interactions have been studied using purified liver plasma membranes and muscular filamentous actin. Despite the large quantity of endogenous actin present in membranes, exogenous muscular filamentous actin cosediments with membranes after a 30 min centrifugation at 30 000 g. The cosedimentation process is time-dependent and exhibits a complex relationship with actin concentration. The cosedimentation of actin with membranes can be partly explained by gelation as shown by low-shear viscosity and electron microscopy. The characterization of the gelation phenomenon as a function of time, actin and membrane concentrations, ionic strength, temperature and Ca2+ concentration is also presented. Gelation alone cannot however account for the overall cosedimentation data, and a more direct mode of association between actin and the membrane must be envisaged. The analogy that exists between the results obtained with liver plasma membranes and those obtained with other membrane systems suggests that a general mechanism may be involved in the interaction of actin with plasma membranes.</p></li>
</ul>
<ul>
<li><a>Burridge K, Kelly T, Connell L</a></li>
<li><strong>Proteins involved in the attachment of actin to the plasma membrane.</strong></li>
<li>Philos Trans R Soc Lond B Biol Sci. 1982; <b>299</b>: 291-9</li>
<li>Display abstract</li>
<li><p>Proteins that may be involved in two types of actin-membrane association are discussed. The first set includes alpha-actinin, vinculin, fimbrin and a new cytoskeletal protein that are all concentrated in adhesion plaques, those regions of cultured fibroblasts where bundles of actin microfilaments terminate and where the plasma membrane comes close to the underlying substrate. The properties of non-muscle alpha-actinin suggest that it functions to cross-link actin filaments and thereby stabilize microfilament bundles rather than functioning in their attachment to the membrane. Fimbrin also appears to be involved in bundling of filaments rather than in attachment. In contrast, vinculin binds to the ends of actin filaments in vitro and is probably the best candidate for a role in the attachment of actin to membranes at the adhesion plaque. The discovery of a new protein, 215k, of unknown function, in the adhesion plaque suggests that many more proteins remain to be identified in this region. Attachment of actin filaments to other regions of the plasma membrane is also considered and a protein is described that seems to be a spectrin in brain and other tissues. The brain protein resembles erythrocyte spectrin in its physical properties, in binding actin, in being associated with cell membranes and in cross-reacting immunologically. We suggest that the brain protein and erythrocyte spectrin both belong to a family of related proteins (the spectrins) which function in the attachment of actin to membranes in many different cell types.</p></li>
</ul>
<ul>
<li><a>Lin S, Cribbs DH, Wilkins JA, Casella JF, Magargal WW, Lin DC</a></li>
<li><strong>The capactins, a class of proteins that cap the ends of actin filaments.</strong></li>
<li>Philos Trans R Soc Lond B Biol Sci. 1982; <b>299</b>: 263-73</li>
<li>Display abstract</li>
<li><p>A number of proteins that bind specifically to the barbed ends of actin filaments in a cytochalasin-like manner have been purified to various degrees from a variety of muscle and non-muscle cells and tissues. Preliminary evidence also indicates that proteins that interact with the pointed ends of filaments are present in skeletal muscle. Because of their ability to cap one or the other end of an actin filament, we have designated this class of proteins as the 'capactins'. On the basis of their effect on actin filament assembly and interaction in vitro, we propose that the capactins play important roles in cellular regulation of actin-based cytoskeletal and contractile functions. Our finding that the disappearance of actin filament bundles in virally transformed fibroblasts can be correlated with an increase in capactin activity in the extracts of these cells is consistent with this hypothesis.</p></li>
</ul>
<ul>
<li><a>Regnouf F, Delobbe A, Gabrion J, Mesnier D, Pradel LA</a></li>
<li><strong>Isolation from thyroid cells or purified plasma membranes with associated actin microfilaments. Proteins bound to actin.</strong></li>
<li>Eur J Biochem. 1982; <b>122</b>: 153-61</li>
<li>Display abstract</li>
<li><p>Plasma membranes of thyroid cells were purified from hog thyroid glands following two procedures. Their homogeneity was tested by electron microscopy and by measurements of the activity of membrane-bound enzyme markers. According to the procedure used the membrane fractions obtained present some differences in their morphological features as well as in the repartition of the activities of the membrane-bound enzyme markers. However, whatever the composition of the membrane fraction examined (membrane vesicles, single membrane sheets with junctional complexes), decoration with heavy meromyosin clearly shows the presence of actin filaments attached to these fragments. Analysis of proteins by polyacrylamide gel electrophoresis indicates the presence of about twelve major components with actin. Treatment of membranes with Triton X-100 results in an insoluble core which contains all the actin and most of the major proteins. The selective extraction of these components by buffers differing in their ionic strength, pH, or the presence or absence of ATP X Mg has been used to characterize some of the proteins associated to actin; among them are filamin, myosin, alpha-actinin, tropomyosin.</p></li>
</ul>
<ul>
<li><a>Wilkins JA, Lin S</a></li>
<li><strong>High-affinity interaction of vinculin with actin filaments in vitro.</strong></li>
<li>Cell. 1982; <b>28</b>: 83-90</li>
<li>Display abstract</li>
<li><p>Immunofluorescence and microinjection experiments have shown that vinculin (molecular weight 130,000) is localized at adhesion plaques of fibroblasts spread on a solid substrate. We found that this protein affects actin filament assembly and interactions in vitro at substoichiometric levels. Vinculin inhibits the rate of actin polymerization under conditions that limit nuclei formation, indicating an effect on the filament elongation step of the reaction. Vinculin also reduces actin filament--filament interactions measured with a low-shear viscometer. Scatchard plot analysis of the binding of 3H-labeled vinculin to actin filaments showed that there is one high-affinity binding site (dissociation constant=20 nM) for every 1,500-2,000 actin monomers. These results suggested that vinculin interacts with a specific site located at the growing ends of actin filaments in a cytochalasin-like manner, a property consistent with its proposed function as a linkage protein between filaments and the plasma membranes.</p></li>
</ul>
<ul>
<li><a>Hartwig JH, Tyler J, Stossel TP</a></li>
<li><strong>Actin-binding protein promotes the bipolar and perpendicular branching of actin filaments.</strong></li>
<li>J Cell Biol. 1980; <b>87</b>: 841-8</li>
<li>Display abstract</li>
<li><p>Branching filaments with striking perpendicularity form when actin polymerizes in the presence of macrophage actin-binding protein. Actin-binding protein molecules are visible at the branch points. Compared with actin polymerized in the absence of actin-binding proteins, not only do the filaments branch but the average length of the actin filaments decreases from 3.2 to 0.63 micrometer. Arrowhead complexes formed by addition of heavy meromyosin molecules to the branching actin filaments point toward the branch points. Actin-binding protein also accelerates the onset of actin polymerization. All of these findings show that actin filaments assemble from nucleating sites on actin-binding protein dimers. A branching polymerization of actin filaments from a preexisting lattice of actin filaments joined by actin-binding protein molecules could generate expansion of cortical cytoplasm in amoeboid cells.</p></li>
</ul>
<ul>
<li><a>Isenberg G, Aebi U, Pollard TD</a></li>
<li><strong>An actin-binding protein from Acanthamoeba regulates actin filament polymerization and interactions.</strong></li>
<li>Nature. 1980; <b>288</b>: 455-9</li>
<li>Display abstract</li>
<li><p>A protein has been purified from Acanthamoeba which, like cytochalasin B, caps the end of actin filaments normally favoured for monomer addition and inhibits the interactions of actin filaments. In addition, this 'capping' protein nucleates the polymerization of actin monomers and blocks the annealing of actin filament fragments.</p></li>
</ul>
<ul>
<li><a>Boxer LA, Richardson S, Floyd A</a></li>
<li><strong>Identification of actin-binding protein in membrane of polymorphonuclear leukocytes.</strong></li>
<li>Nature. 1976; <b>263</b>: 249-51</li>
</ul>
</div></li>
<li><span>Metabolism</span> (metabolic pathways involving proteins which contain this domain)</li>
<li>
<table><tbody><tr><td><table><tbody><tr><th>% proteins involved</th><th>KEGG pathway ID</th><th>Description</th></tr><tr><td>90.20</td><td> <a>map04110</a></td><td>Cell cycle</td></tr><tr><td>9.80</td><td> <a>map04115</a></td><td>p53 signaling pathway</td></tr></tbody></table><p>This information is based on mapping of SMART genomic protein database to <a>KEGG</a> orthologous groups. Percentage points are related to the number of proteins with 14_3_3 domain which could be assigned to a KEGG orthologous group, and not all proteins containing 14_3_3 domain. Please note that proteins can be included in multiple pathways, ie. the numbers above will not always add up to 100%.</p></td></tr></tbody></table></li><li><span>Structure</span> (3D structures containing this domain)</li><li><h3>3D Structures of 14_3_3 domains in <a>PDB</a></h3><table><tbody><tr><th>PDB code</th><th>Main view</th><th>Title</th></tr><tr><td><a>1a37</a></td><td>14-3-3 PROTEIN ZETA BOUND TO PS-RAF259 PEPTIDE</td></tr> <tr><td><a>1a38</a></td><td>14-3-3 PROTEIN ZETA BOUND TO R18 PEPTIDE</td></tr> <tr><td><a>1a4o</a></td><td>14-3-3 PROTEIN ZETA ISOFORM</td></tr> <tr><td><a>1ib1</a></td><td>CRYSTAL STRUCTURE OF THE 14-3-3 ZETA:SEROTONIN N-ACETYLTRANSFERASE COMPLEX</td></tr> <tr><td><a>1o9c</a></td><td>Structural view of a fungal toxin acting on a 14-3-3 regulatory complex</td></tr> <tr><td><a>1o9d</a></td><td>Structural view of a fungal toxin acting on a 14-3-3 regulatory complex</td></tr> <tr><td><a>1o9e</a></td><td>Structural view of a fungal toxin acting on a 14-3-3 regulatory complex</td></tr> <tr><td><a>1o9f</a></td><td>Structural view of a fungal toxin acting on a 14-3-3 regulatory complex</td></tr> <tr><td><a>1qja</a></td><td>14-3-3 ZETA/PHOSPHOPEPTIDE COMPLEX (MODE 2)</td></tr> <tr><td><a>1qjb</a></td><td>14-3-3 ZETA/PHOSPHOPEPTIDE COMPLEX (MODE 1)</td></tr> <tr><td><a>1ywt</a></td><td>Crystal structure of the human sigma isoform of 14-3-3 in complex with a mode-1 phosphopeptide</td></tr> <tr><td><a>1yz5</a></td><td>The crystal structure of 14-3-3-sigma at 2.8 angstrom resolution</td></tr> <tr><td><a>2b05</a></td><td>Crystal Structure of 14-3-3 gamma in complex with a phosphoserine peptide</td></tr> <tr><td><a>2bq0</a></td><td>14-3-3 Protein Beta (Human)</td></tr> <tr><td><a>2br9</a></td><td>14-3-3 Protein Epsilon (Human) Complexed to Peptide</td></tr> <tr><td><a>2btp</a></td><td>14-3-3 Protein Theta (Human) Complexed to Peptide</td></tr> <tr><td><a>2c1j</a></td><td>Molecular basis for the recognition of phosphorylated and  phosphoacetylated histone H3 by 14-3-3</td></tr> <tr><td><a>2c1n</a></td><td>Molecular basis for the recognition of phosphorylated and  phosphoacetylated histone H3 by 14-3-3</td></tr> <tr><td><a>2c23</a></td><td>14-3-3 Protein Beta (Human) in complex with exoenzyme S peptide</td></tr> <tr><td><a>2c63</a></td><td>14-3-3 Protein Eta (Human) Complexed to Peptide</td></tr> <tr><td><a>2c74</a></td><td>14-3-3 Protein Eta (Human) Complexed to Peptide</td></tr> <tr><td><a>2npm</a></td><td>crystal structure of Cryptosporidium parvum 14-3-3 protein in complex with peptide</td></tr> <tr><td><a>2o02</a></td><td>Phosphorylation independent interactions between 14-3-3 and Exoenzyme S: from structure to pathogenesis</td></tr> <tr><td><a>2o98</a></td><td>Structure of the 14-3-3 / H+-ATPase plant complex</td></tr> <tr><td><a>2v7d</a></td><td>14-3-3 protein zeta in complex with Thr758 phosphorylated integrin  beta2 peptide</td></tr> <tr><td><a>2wh0</a></td><td>Recognition of an intrachain tandem 14-3-3 binding site within  protein kinase C epsilon</td></tr> <tr><td><a>3axy</a></td><td>Structure of Florigen Activation Complex Consisting of Rice Florigen Hd3a, 14-3-3 Protein GF14 and Rice FD Homolog OsFD1</td></tr> <tr><td><a>3cu8</a></td><td>Impaired binding of 14-3-3 to Raf1 is linked to Noonan and LEOPARD syndrome</td></tr> <tr><td><a>3e6y</a></td><td>Structure of 14-3-3 in complex with the differentiation-inducing agent Cotylenin A</td></tr> <tr><td><a>3efz</a></td><td>Crystal Structure of a 14-3-3 protein from cryptosporidium parvum (cgd1_2980)</td></tr> <tr><td><a>3iqj</a></td><td>Crystal Structure of human 14-3-3 sigma in Complex with Raf1 peptide (10mer)</td></tr> <tr><td><a>3iqu</a></td><td>Crystal Structure of human 14-3-3 sigma in Complex with Raf1 peptide (6mer)</td></tr> <tr><td><a>3iqv</a></td><td>Crystal Structure of human 14-3-3 sigma in Complex with Raf1 peptide (6mer) and stabilisator Fusicoccin</td></tr> <tr><td><a>3lw1</a></td><td>Binary complex of 14-3-3 sigma and p53 pT387-peptide</td></tr> <tr><td><a>3m50</a></td><td>Structure of the 14-3-3/PMA2 complex stabilized by Epibestatin</td></tr> <tr><td><a>3m51</a></td><td>Structure of the 14-3-3/PMA2 complex stabilized by Pyrrolidone1</td></tr> <tr><td><a>3mhr</a></td><td>14-3-3 sigma in complex with YAP pS127-peptide</td></tr> <tr><td><a>3nkx</a></td><td>Impaired binding of 14-3-3 to Raf1 is linked to Noonan and LEOPARD syndrome</td></tr> <tr><td><a>3o8i</a></td><td>Structure of 14-3-3 isoform sigma in complex with a C-Raf1 peptide and a stabilizing small molecule fragment</td></tr> <tr><td><a>3p1n</a></td><td>Crystal structure of human 14-3-3 sigma in complex with TASK-3 peptide</td></tr> <tr><td><a>3p1o</a></td><td>Crystal structure of human 14-3-3 sigma in complex with TASK-3 peptide and stabilisator Fusicoccin A</td></tr> <tr><td><a>3p1p</a></td><td>Crystal structure of human 14-3-3 sigma C38N/N166H in complex with TASK-3 peptide</td></tr> <tr><td><a>3p1q</a></td><td>Crystal structure of human 14-3-3 sigma C38N/N166H in complex with TASK-3 Peptide and stabilizer fusicoccin A</td></tr> <tr><td><a>3p1r</a></td><td>Crystal structure of human 14-3-3 sigma C38V/N166H in complex with TASK-3 peptide</td></tr> <tr><td><a>3p1s</a></td><td>Crystal structure of human 14-3-3 sigma C38N/N166H in complex with TASK-3 peptide and stabilizer fusicoccin A</td></tr> <tr><td><a>3rdh</a></td><td>X-ray induced covalent inhibition of 14-3-3</td></tr> <tr><td><a>3smk</a></td><td>Crystal structure of human 14-3-3 sigma C38V/N166H in complex with TASK-3 peptide and stabilizer Cotylenin A</td></tr> <tr><td><a>3sml</a></td><td>Crystal structure of human 14-3-3 sigma C38N/N166H in complex with TASK-3 peptide and stabilizer Fusicoccin A aglycone</td></tr> <tr><td><a>3smm</a></td><td>Crystal structure of human 14-3-3 sigma C38N/N166H in complex with task-3 peptide and stabilizer Fusicoccin J aglycone</td></tr> <tr><td><a>3smn</a></td><td>Crystal structure of human 14-3-3 sigma C38N/N166H in complex with task-3 peptide and stabilizer Fusicoccin A-THF</td></tr> <tr><td><a>3smo</a></td><td>Crystal structure of human 14-3-3 sigma C38V/N166H in complex with TASK-3 peptide and stabilizer Fusicoccin J aglycone</td></tr> <tr><td><a>3sp5</a></td><td>3SP5</td></tr> <tr><td><a>3spr</a></td><td>Crystal structure of human 14-3-3 sigma C38V/N166H in complex with TASK-3 peptide and stabilizer FC-THF</td></tr> <tr><td><a>3t0l</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>3t0m</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>3u9x</a></td><td>Covalent attachment of pyridoxal-phosphate derivatives to 14-3-3 proteins</td></tr> <tr><td><a>3ual</a></td><td>Crystal Structure of 14-3-3 epsilon with Mlf1 peptide</td></tr> <tr><td><a>3ubw</a></td><td>Complex of 14-3-3 isoform epsilon, a Mlf1 phosphopeptide and a small fragment hit from a FBDD screen</td></tr> <tr><td><a>3ux0</a></td><td>Crystal structure of human 14-3-3 sigma in complex with TASK-3 peptide and stabilizer Fusicoccin H</td></tr> <tr><td><a>3uzd</a></td><td>Crystal structure of 14-3-3 GAMMA</td></tr> <tr><td><a>4bg6</a></td><td>14-3-3 interaction with Rnd3 prenyl-phosphorylation motif</td></tr> <tr><td><a>4dat</a></td><td>Structure of 14-3-3 sigma in complex with PADI6 14-3-3 binding motif II</td></tr> <tr><td><a>4dau</a></td><td>Structure of 14-3-3 sigma in complex with PADI6 14-3-3 binding motif I</td></tr> <tr><td><a>4dhm</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>4dhn</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>4dho</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>4dhp</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>4dhq</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>4dhr</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>4dhs</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>4dht</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>4dhu</a></td><td>Small-molecule inhibitors of 14-3-3 protein-protein interactions from virtual screening</td></tr> <tr><td><a>4dnk</a></td><td>Crystal structure of a tyrosine 3-monooxygenase/tryptophan 5-monooxygenase activation protein, beta polypeptide (YWHAB) from homo sapiens at 2.20 A resolution.</td></tr> <tr><td><a>4dx0</a></td><td>Structure of the 14-3-3/PMA2 complex stabilized by a pyrazole derivative</td></tr> <tr><td><a>4e2e</a></td><td>Crystal structure of a tyrosine 3-monooxygenase/tryptophan 5-monooxygenase activation protein, gamma polypeptide (YWHAG) from Homo sapiens at 2.25 A resolution</td></tr> <tr><td><a>4f7r</a></td><td>Crystal structure of 14-3-3 protein from Giardia intestinalis</td></tr> <tr><td><a>4fj3</a></td><td>14-3-3 isoform zeta in complex with a diphoyphorylated C-RAF peptide</td></tr> <tr><td><a>4fl5</a></td><td>Crystal structure of human 14-3-3 sigma in complex with a Tau-protein peptide surrounding pS214</td></tr> <tr><td><a>4fr3</a></td><td>Crystal structure of human 14-3-3 sigma in complex with TASK-3 peptide and stabilizer 16-O-Me-FC-H</td></tr> <tr><td><a>4gnt</a></td><td>Complex of ChREBP and 14-3-3beta</td></tr> <tr><td><a>4hkc</a></td><td>14-3-3-zeta in complex with S1011 phosphorylated integrin alpha-4 peptide</td></tr> <tr><td><a>4hqw</a></td><td>Molecular tweezers modulate 14-3-3 protein-protein interactions</td></tr> <tr><td><a>4hru</a></td><td>Molecular tweezers modulate 14-3-3 protein-protein interactions</td></tr> <tr><td><a>4iea</a></td><td>14-3-3 isoform sigma in complex with a phosphorylated C-RAF peptide</td></tr> <tr><td><a>4ihl</a></td><td>Human 14-3-3 isoform zeta in complex with a diphoyphorylated C-RAF peptide and Cotylenin A</td></tr> <tr><td><a>4j6s</a></td><td>14-3-3gamma complexed with the N-terminal sequence of tyrosine hydroxylase (residues 1-43)</td></tr> <tr><td><a>4jc3</a></td><td>14-3-3 protein interaction with Estrogen Receptor Alpha provides a novel drug target interface</td></tr> <tr><td><a>4jdd</a></td><td>14-3-3 protein interaction with Estrogen Receptor Alpha provides a novel drug target interface</td></tr> <tr><td><a>4n7g</a></td><td>Crystal structure of 14-3-3zeta in complex with a peptide derived from ExoS</td></tr> <tr><td><a>4n7y</a></td><td>Crystal structure of 14-3-3zeta in complex with a 8-carbon-linker cyclic peptide derived from ExoS</td></tr> <tr><td><a>4n84</a></td><td>Crystal structure of 14-3-3zeta in complex with a 12-carbon-linker cyclic peptide derived from ExoS</td></tr> <tr><td><a>4o46</a></td><td>14-3-3-gamma in complex with influenza NS1 C-terminal tail phosphorylated at S228</td></tr> <tr><td><a>4qli</a></td><td>4QLI</td></tr> <tr><td><a>4wrq</a></td><td>4WRQ</td></tr> <tr><td><a>4y32</a></td><td>4Y32</td></tr> <tr><td><a>4y3b</a></td><td>4Y3B</td></tr> <tr><td><a>4y5i</a></td><td>4Y5I</td></tr> <tr><td><a>4zdr</a></td><td>4ZDR</td></tr> <tr><td><a>4zq0</a></td><td>4ZQ0</td></tr> <tr><td><a>5btv</a></td><td>5BTV</td></tr> <tr><td><a>5by9</a></td><td>5BY9</td></tr> <tr><td><a>5d2d</a></td><td>5D2D</td></tr> <tr><td><a>5d3e</a></td><td>5D3E</td></tr> <tr><td><a>5d3f</a></td><td>5D3F</td></tr> <tr><td><a>5ewz</a></td><td>5EWZ</td></tr> <tr><td><a>5exa</a></td><td>5EXA</td></tr> <tr><td><a>5f74</a></td><td>5F74</td></tr> <tr><td><a>5hf3</a></td><td>5HF3</td></tr> <tr><td><a>5iqp</a></td><td>5IQP</td></tr> <tr><td><a>5j31</a></td><td>5J31</td></tr> <tr><td><a>5lho</a></td><td>5LHO</td></tr> <tr><td><a>5lvz</a></td><td>5LVZ</td></tr> <tr><td><a>5lx2</a></td><td>5LX2</td></tr> </tbody></table></li>
<li><span>Links</span> (links to other resources describing this domain)</li>
<li>
<table>
<tbody><tr><th>INTERPRO</th><td><a>IPR023410</a></td></tr><tr><th>PFAM</th><td><a>14-3-3</a></td></tr></tbody></table></li></ul><table>
<tbody><tr><th><h2>DAX</h2>Domain present in Dishevelled and axin</th>
<td><img alt="DAX"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00021
    </td>
</tr><tr><th>Description:</th>
<td>Domain of unknown function.</td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="DAX"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>DEATH</h2>DEATH domain, found in proteins involved in cell death (apoptosis).</th>
<td><img alt="DEATH"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00005
    </td>
</tr><tr><th>Description:</th>
<td>Alpha-helical domain present in a variety of proteins with apoptotic functions. Some (but not all) of these domains form homotypic and heterotypic dimers.</td></tr>

<tr><th>GO process:</th><td>signal transduction (<a>GO:0007165</a>)</td></tr>
<tr><th>GO function:</th><td>protein binding (<a>GO:0005515</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="DEATH"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>DED</h2>Death effector domain</th>
<td><img alt="DED"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00031
    </td>
</tr><tr><th>Description:</th>
<td> - </td></tr>

<tr><th>GO process:</th><td>regulation of apoptotic process (<a>GO:0042981</a>)</td></tr>
<tr><th>GO function:</th><td>protein binding (<a>GO:0005515</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="DED"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>DEP</h2>Domain found in Dishevelled, Egl-10, and Pleckstrin</th>
<td><img alt="DEP"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00049
    </td>
</tr><tr><th>Description:</th>
<td>Domain of unknown function present in signalling proteins that contain PH, rasGEF, rhoGEF, rhoGAP, RGS, PDZ domains. DEP domain in Drosophila dishevelled is essential to rescue planar polarity defects and induce JNK signalling (Cell 94, 109-118).</td></tr>

<tr><th>GO process:</th><td>intracellular signal transduction (<a>GO:0035556</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="DEP"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>DSPc</h2>Dual specificity phosphatase, catalytic domain</th>
<td><img alt="DSPc"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00195
    </td>
</tr><tr><th>Description:</th>
<td> - </td></tr>

<tr><th>GO process:</th><td>protein dephosphorylation (<a>GO:0006470</a>)</td></tr>
<tr><th>GO function:</th><td>protein tyrosine/serine/threonine phosphatase activity (<a>GO:0008138</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="DSPc"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>DYNc</h2>Dynamin, GTPase</th>
<td><img alt="DYNc"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00053
    </td>
</tr><tr><th>Description:</th>
<td>Large GTPases that mediate vesicle trafficking. Dynamin participates in the endocytic uptake of receptors, associated ligands, and  plasma membrane following an exocytic event.</td></tr>

<tr><th>GO function:</th><td>GTPase activity (<a>GO:0003924</a>), GTP binding (<a>GO:0005525</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="DYNc"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>EAL</h2>Putative diguanylate phosphodiesterase</th>
<td><img alt="EAL"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00052
    </td>
</tr><tr><th>Description:</th>
<td>Putative diguanylate phosphodiesterase, present in a variety of bacteria.</td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="EAL"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>EFh</h2>EF-hand, calcium binding motif</th>
<td><img alt="EFh"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00054
    </td>
</tr><tr><th>Description:</th>
<td>EF-hands are calcium-binding motifs that occur at least in pairs. Links between disease states and genes encoding EF-hands, particularly the S100 subclass, are emerging. Each motif consists of a 12 residue loop flanked on either side by a 12 residue alpha-helix. EF-hands undergo a conformational change unpon binding calcium ions.</td></tr>

<tr><th>GO function:</th><td>calcium ion binding (<a>GO:0005509</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="EFh"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>EH</h2>Eps15 homology domain</th>
<td><img alt="EH"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00027
    </td>
</tr><tr><th>Description:</th>
<td>Pair of EF hand motifs that recognise proteins containing Asn-Pro-Phe (NPF) sequences.</td></tr>

<tr><th>GO function:</th><td>protein binding (<a>GO:0005515</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="EH"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>ENTH</h2>Epsin N-terminal homology (ENTH) domain</th>
<td><img alt="ENTH"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00273
    </td>
</tr><tr><th>Description:</th>
<td> - </td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="ENTH"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>FBOX</h2>A Receptor for Ubiquitination Targets</th>
<td><img alt="FBOX"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00256
    </td>
</tr><tr><th>Description:</th>
<td> - </td></tr>

<tr><th>GO function:</th><td>protein binding (<a>GO:0005515</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="FBOX"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>FCH</h2>Fes/CIP4 homology domain</th>
<td><img alt="FCH"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00055
    </td>
</tr><tr><th>Description:</th>
<td>Alignment extended from original report. Highly alpha-helical. Also known as the RAEYL motif or the S. pombe Cdc15 N-terminal domain.</td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="FCH"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>FF</h2>Contains two conserved F residues</th>
<td><img alt="FF"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00441
    </td>
</tr><tr><th>Description:</th>
<td>A novel motif that often accompanies WW domains. Often contains two conserved Phe (F) residues. </td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="FF"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>FH2</h2>Formin Homology 2 Domain</th>
<td><img alt="FH2"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00498
    </td>
</tr><tr><th>Description:</th>
<td>FH proteins control rearrangements of the actin cytoskeleton, especially in the context of cytokinesis and cell polarisation. Members of this family have been found to interact with Rho-GTPases, profilin and other actin-assoziated proteins. These interactions are mediated by the proline-rich FH1 domain, usually located in front of FH2 (but not listed in SMART). Despite this cytosolic function, vertebrate formins have been assigned functions within the nucleus. A set of Formin-Binding Proteins (FBPs) has been shown to bind FH1 with their WW domain.</td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="FH2"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>FHA</h2>Forkhead associated domain</th>
<td><img alt="FHA"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00240
    </td>
</tr><tr><th>Description:</th>
<td>Found in eukaryotic and prokaryotic proteins. Putative nuclear signalling domain.</td></tr>

<tr><th>GO function:</th><td>protein binding (<a>GO:0005515</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="FHA"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>FIST</h2>FIST N domain</th>
<td><img alt="FIST"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00897
    </td>
</tr><tr><th>Description:</th>
<td>The FIST N domain is a novel sensory domain, which is present in signal transduction proteins from Bacteria, Archaea and Eukarya. Chromosomal proximity of FIST-encoding genes to those coding for proteins involved in amino acid metabolism and transport suggest that FIST domains bind small ligands, such as amino acids.</td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="FIST"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>Frizzled</h2>Frizzled/Smoothened family membrane region</th>
<td><img alt="Frizzled"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM01330
    </td>
</tr><tr><th>Description:</th>
<td>Frizzled is a family of G protein-coupled receptor proteins (PMID: 14977528) that serves as receptors in the Wnt signaling pathway and other signaling pathways. When activated, Frizzled leads to activation of Dishevelled in the cytosol.</td></tr>

<tr><th>GO process:</th><td>cell surface receptor signaling pathway (<a>GO:0007166</a>)</td></tr>
<tr><th>GO component:</th><td>membrane (<a>GO:0016020</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="Frizzled"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>FYVE</h2>Protein present in Fab1, YOTB, Vac1, and EEA1</th>
<td><img alt="FYVE"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00064
    </td>
</tr><tr><th>Description:</th>
<td>The FYVE zinc finger is named after four proteins where it was first found: Fab1, YOTB/ZK632.12, Vac1, and EEA1. The FYVE finger has been shown to bind two Zn2+ ions. The FYVE finger has eight potential zinc coordinating cysteine positions. The FYVE finger is structurally related to the <a>PHD finger</a> and the <a>RING finger</a>. Many members of this family also include two histidines in a motif R+HHC+XCG, where + represents a charged residue and X any residue.  The FYVE finger functions in the membrane recruitment of cytosolic proteins by binding to phosphatidylinositol 3-phosphate (PI3P), which is prominent on endosomes. The R+HHC+XCG motif is critical for PI3P binding.</td></tr>

<tr><th>GO function:</th><td>metal ion binding (<a>GO:0046872</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="FYVE"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>G_alpha</h2>G protein alpha subunit</th>
<td><img alt="G_alpha"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00275
    </td>
</tr><tr><th>Description:</th>
<td>Subunit of G proteins that contains the guanine nucleotide binding site</td></tr>

<tr><th>GO process:</th><td>G protein-coupled receptor signaling pathway (<a>GO:0007186</a>)</td></tr>
<tr><th>GO function:</th><td>GTPase activity (<a>GO:0003924</a>), guanyl nucleotide binding (<a>GO:0019001</a>), G-protein beta/gamma-subunit complex binding (<a>GO:0031683</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="G_alpha"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>GAF</h2>Domain present in phytochromes and cGMP-specific phosphodiesterases.</th>
<td><img alt="GAF"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00065
    </td>
</tr><tr><th>Description:</th>
<td>Mutations within these domains in PDE6B result in autosomal recessive  inheritance of retinitis pigmentosa. </td></tr>

<tr><th>GO function:</th><td>protein binding (<a>GO:0005515</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="GAF"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>GAS2</h2>Growth-Arrest-Specific Protein 2 Domain</th>
<td><img alt="GAS2"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00243
    </td>
</tr><tr><th>Description:</th>
<td>GROWTH-ARREST-SPECIFIC PROTEIN 2 Domain</td></tr>

<tr><th>GO function:</th><td>microtubule binding (<a>GO:0008017</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="GAS2"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>GED</h2>Dynamin GTPase effector domain</th>
<td><img alt="GED"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00302
    </td>
</tr><tr><th>Description:</th>
<td> - </td></tr>

<tr><th>GO function:</th><td>GTPase activity (<a>GO:0003924</a>), GTP binding (<a>GO:0005525</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="GED"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>GGDEF</h2>diguanylate cyclase</th>
<td><img alt="GGDEF"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00267
    </td>
</tr><tr><th>Description:</th>
<td>Diguanylate cyclase, present in a variety of bacteria.</td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="GGDEF"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>GGL</h2>G protein gamma subunit-like motifs</th>
<td><img alt="GGL"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00224
    </td>
</tr><tr><th>Description:</th>
<td> - </td></tr>

<tr><th>GO process:</th><td>G protein-coupled receptor signaling pathway (<a>GO:0007186</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="GGL"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>GoLoco</h2>LGN motif, putative GEFs specific for G-alpha GTPases</th>
<td><img alt="GoLoco"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00390
    </td>
</tr><tr><th>Description:</th>
<td>GEF specific for Galpha_i proteins</td></tr>

<tr><th>GO function:</th><td>GTPase regulator activity (<a>GO:0030695</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="GoLoco"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>GRAM</h2>domain in glucosyltransferases, myotubularins and other putative membrane-associated proteins</th>
<td><img alt="GRAM"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00568
    </td>
</tr><tr><th>Description:</th>
<td> - </td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="GRAM"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>GS</h2>GS motif</th>
<td><img alt="GS"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00467
    </td>
</tr><tr><th>Description:</th>
<td>Aa approx. 30 amino acid motif that precedes the kinase domain in types I and II TGF beta receptors. Mutation of two or more of the serines or threonines in the TTSGSGSG of TGF-beta type I receptor impairs phosphorylation and signaling activity.</td></tr>

<tr><th>GO process:</th><td>protein phosphorylation (<a>GO:0006468</a>)</td></tr>
<tr><th>GO component:</th><td>membrane (<a>GO:0016020</a>)</td></tr>
<tr><th>GO function:</th><td>ATP binding (<a>GO:0005524</a>), transmembrane receptor protein serine/threonine kinase activity (<a>GO:0004675</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="GS"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>GuKc</h2>Guanylate kinase homologues.</th>
<td><img alt="GuKc"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00072
    </td>
</tr><tr><th>Description:</th>
<td>Active enzymes catalyze ATP-dependent phosphorylation of GMP to GDP. Structure resembles that of adenylate kinase. So-called membrane-associated guanylate kinase homologues (MAGUKs) do not possess guanylate kinase activities; instead at least some possess protein-binding functions.</td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="GuKc"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>HAMP</h2>HAMP (Histidine kinases, Adenylyl cyclases, Methyl binding proteins, Phosphatases) domain</th>
<td><img alt="HAMP"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00304
    </td>
</tr><tr><th>Description:</th>
<td> - </td></tr>

<tr><th>GO process:</th><td>signal transduction (<a>GO:0007165</a>)</td></tr>
<tr><th>GO component:</th><td>integral component of membrane (<a>GO:0016021</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="HAMP"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>HATPase_c</h2>Histidine kinase-like ATPases</th>
<td><img alt="HATPase_c"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00387
    </td>
</tr><tr><th>Description:</th>
<td>Histidine kinase-, DNA gyrase B-, phytochrome-like ATPases.</td></tr>

<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="HATPase_c"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>HECTc</h2>Domain Homologous to E6-AP Carboxyl Terminus with </th>
<td><img alt="HECTc"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00119
    </td>
</tr><tr><th>Description:</th>
<td>E3 ubiquitin-protein ligases. Can bind to E2 enzymes.</td></tr>

<tr><th>GO function:</th><td>ubiquitin-protein transferase activity (<a>GO:0004842</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="HECTc"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>HisKA</h2>His Kinase A (phosphoacceptor) domain</th>
<td><img alt="HisKA"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00388
    </td>
</tr><tr><th>Description:</th>
<td>Dimerisation and phosphoacceptor domain of histidine kinases. </td></tr>

<tr><th>GO process:</th><td>signal transduction (<a>GO:0007165</a>)</td></tr>
<tr><th>GO function:</th><td>phosphorelay sensor kinase activity (<a>GO:0000155</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="HisKA"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>HPT</h2>Histidine Phosphotransfer domain</th>
<td><img alt="HPT"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00073
    </td>
</tr><tr><th>Description:</th>
<td>Contains an active histidine residue that mediates phosphotransfer reactions. Domain detected only in eubacteria. This alignment is an extension to that shown in the Cell structure paper.</td></tr>

<tr><th>GO process:</th><td>phosphorelay signal transduction system (<a>GO:0000160</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="HPT"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>Hr1</h2>Rho effector or protein kinase C-related kinase homology region 1 homologues</th>
<td><img alt="Hr1"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00742
    </td>
</tr><tr><th>Description:</th>
<td>Alpha-helical domain found in vertebrate PRK1 and yeast PKC1 protein kinases C. The HR1 in rhophilin bind RhoGTP; those in PRK1 bind RhoA and RhoB. Also called RBD - Rho-binding domain </td></tr>

<tr><th>GO process:</th><td>signal transduction (<a>GO:0007165</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="Hr1"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table><table>
<tbody><tr><th><h2>ILWEQ</h2>I/LWEQ domain</th>
<td><img alt="ILWEQ"/></td></tr></tbody></table><table>
<tbody><tr><th>SMART accession number:</th><td>SM00307
    </td>
</tr><tr><th>Description:</th>
<td>Thought to possess an F-actin binding function.</td></tr>

<tr><th>GO function:</th><td>actin binding (<a>GO:0003779</a>)</td></tr>
<tr><th>Family alignment:</th><td>
<form>
<input name="NAME" value="ILWEQ"/>
    View <input name="DO_CONSENSUS" type="submit" value="Alignment consensus sequence"/> or 
    <input name="SHOW_ALIGNMENT" type="submit" value="Family alignment in"/>
<select name="FORMAT"><option value="CHROMA">CHROMA format</option><option value="LOGO">HMM/Web logo</option><option value="CLUSTAL">CLUSTALW format</option><option value="MSF">MSF format</option><option value="FASTA">FASTA format</option><option value="PIR">PIR format</option>
</select></form></td>
</tr></tbody></table></div><div>
<div>© 2020 EMBL | <a>Privacy Policy</a></div>Send comments to <a>Ivica Letunic</a></div>
</body></html>